src/app/billingentity/billingentity-members/billing-entity-members.component.ts

```diff
@@ -258,19 +255,15 @@ export class BillingEntityMembersComponent implements OnInit, OnDestroy {
     });
     forkJoin(upsert$).subscribe({
       next: () => {
-        this.messageService.add({
-          severity: 'success',
-          summary: $localize`Successfully saved`,
-        });
+        this.notificationService.showSuccessMessage(
+          $localize`Successfully saved Billing ${payload.billingEntity.metadata.name}.`
+        );
         void this.router.navigate([this.navigationService.previousLocation()], { relativeTo: this.route });
       },
-      error: (error) => {
-        this.messageService.add({
-          severity: 'error',
-          summary: $localize`Error`,
-          detail: error.message,
-          sticky: true,
-        });
+      error: () => {
+        this.notificationService.showErrorMessage(
+          $localize`Could not save Billing ${payload.billingEntity.metadata.name}. Please try again later.`
```

Comment:
(also consider using display name here.)

Nit discussion: Are we sure "try again later" actually helps in every case?
What if the error is not something that vanishes just by trying again?
The error might be something that needs admin support, is user's fault or not actually recoverable.
e.g. the user is trying to edit an entity, but someone/something else deleted the entity while the user fills the form, and then the resource is gone (404) when saving, then "try later again" doesn't help.

Maybe remove "Please try again later" and/or add the original error message? I know displaying technical errors might not be user-friendly. However, the users on the Portal are probably quite often developers themselves, who know http error codes :)
Maybe the additional error can be styled in a way that is small, or only visible in an Accordion element or something.

Reply:
Good point regarding the `try again later`, I was a bit torn on that, and you are right, it might not always solve the problem. 

I would rather keep any error status codes & messages out of the toasts, as they are hard to read and in most cases don't help the user (e.g. even a 404 could mean multiple things). It was also a specific request in the ticket to keep any urls & ports out of error toasts.

For the very tech-savy ones, there's still the browser console, and for the cloud-portal/-api devs also the logs and glitchtip.
Wherever possible, we should prevent user errors, e.g. with good validation. And where it makes sense, show a specific, user-friendly message for some error types, e.g. on 409 tell them that this entity already exists. Both of that we already do.